dist/index.js

```diff
@@ -57,13 +57,8 @@ function koaDevware(compiler, options) {
 
   function middleware(context, next) {
     return new _promise2.default(function (resolve, reject) {
-      // https://github.com/webpack/docs/wiki/plugins#donestats-stats
-      compiler.plugin('done', function (stats) {
-        resolve(stats);
-      });
-
-      compiler.plugin('failed', function (error) {
```

Comment:
a non-existing resource in which context? an endpoint/file on the server, or a resource within a file on the server?